src/CabalHelper/Runtime/Main.hs

```diff
@@ -19,7 +19,9 @@
 #undef CH_MIN_VERSION_Cabal
 #define CH_MIN_VERSION_Cabal MIN_VERSION_Cabal
 #endif
-
+#if CH_MIN_VERSION_Cabal(2,0,0)
```

Comment:
Conditionals in the imports go at the end of the list and should be merged into a single block per Cabal version.